lib/internal/languages/languageEn.dart

```diff
@@ -162,69 +162,69 @@ class LanguageEn extends Languages {
   @override
   String get labelLicenses => "Licenses";
   @override
-  String get labelChooseColor => "Choose Color";
+  String get labelChooseColor => "Choose Colour";
   @override
   String get labelTheme => "Theme";
   @override
-  String get labelUseSystemTheme => "Use System Theme";
+  String get labelUseSystemTheme => "System";
   @override
   String get labelUseSystemThemeJustification =>
-    "Enable/Disable automatic Theme";
+    "Automatic Theme";
   @override
-  String get labelEnableDarkTheme => "Enable Dark Theme";
+  String get labelEnableDarkTheme => "Dark";
   @override
   String get labelEnableDarkThemeJustification =>
-    "Use Dark Theme by default";
+    "Dark by default";
   @override
-  String get labelEnableBlackTheme => "Enable Black Theme";
+  String get labelEnableBlackTheme => "Black";
   @override
   String get labelEnableBlackThemeJustification =>
-    "Enable Pure Black Theme";
+    "Pure Black";
   @override
   String get labelAccentColor => "Accent Color";
   @override
   String get labelAccentColorJustification => "Customize accent color";
   @override
   String get labelAudioFolder => "Audio Folder";
   @override
-  String get labelAudioFolderJustification => "Choose a Folder for " +
-    "Audio downloads";
+  String get labelAudioFolderJustification => "Choose a folder for " +
+    "audio downloads";
   @override
   String get labelVideoFolder => "Video Folder";
   @override
   String get labelVideoFolderJustification => "Choose a folder for " +
-    "Video downloads";
+    "video downloads";
   @override
   String get labelAlbumFolder => "Album Folder";
   @override
-  String get labelAlbumFolderJustification => "Create a Folder for each Song Album";
+  String get labelAlbumFolderJustification => "Create a folder for each song album";
   @override
   String get labelDeleteCache => "Delete Cache";
   @override
-  String get labelDeleteCacheJustification => "Clear SongTube Cache";
+  String get labelDeleteCacheJustification => "Clear the SongTube cache";
   @override
   String get labelAndroid11Fix => "Android 11 Fix";
   @override
-  String get labelAndroid11FixJustification => "Fixes Download issues on " +
-    "Android 10 & 11";
+  String get labelAndroid11FixJustification => "Fixes downloading issues on " +
+    "Android 10 and 11";
   @override
   String get labelBackup => "Backup";
   @override
-  String get labelBackupJustification => "Backup your media library";
+  String get labelBackupJustification => "Back up your media library";
```

Comment:
@Clashkid155 If this shouldn't be changed. Is it actually "Backup of your media library"? "Back up *" is a verb. "Backup" is a noun.

Reply:
I see what you mean, carry on.